modules/ROOT/pages/munit-extensions-maven-plugin-configuration.adoc

```diff
@@ -234,18 +228,16 @@ To run tests with specific tags, add a line for the test with the specific tag:
       <artifactId>munit-extensions-maven-plugin</artifactId>
       <configuration>
       ...
-      <munitTags>exampleMunitTag</munitTags>
+      <munitTags>exampleMunitTag,exampleMunitTag2</munitTags>
```

Comment:
This is an example and not really the syntax. I would add a "for example" to the lead-in and would show at least two tags so you can show them separated by commas.